api/app/tests/medium/routers/test_topics.py

```diff
@@ -163,6 +167,42 @@ def test_create_wrong_threat_level_topic():
         create_topic(USER1, _topic)
 
 
+def test_create_wrong_safety_impact_topic():
+    create_user(USER1)
+    _topic = TOPIC1.copy()
+    _topic["safety_impact"] = "test"
+    with pytest.raises(HTTPError, match="422: Unprocessable Entity"):
+        create_topic(USER1, _topic)
+
+
+def test_create_wrong_exploitation_topic():
+    create_user(USER1)
+    _topic = TOPIC1.copy()
+    _topic["exploitation"] = "test"
+    with pytest.raises(HTTPError, match="422: Unprocessable Entity"):
+        create_topic(USER1, _topic)
+
+
+def test_create_topic_default_value_when_registered_empty():
```

Comment:
```suggestion
def test_default_value_is_set_when_ssvc_related_value_is_empty_in_creation():
```

いい名前が浮かばなかったですが、確かめたいことは「ssvcに関連する値が作成時に空の場合、デフォルト値がセットされている」かなと